src/tests/functional/plugin/shared/include/snippets/add.hpp

```diff
@@ -18,13 +18,28 @@ typedef std::tuple<
         std::string                  // Target Device
 > AddParams;
 
+typedef std::tuple<
+        std::vector<ov::Shape>,      // Input 0, Input 1 Shape
+        size_t,                      // Expected num nodes
+        size_t,                      // Expected num subgraphs
+        std::string                  // Target Device
+> AddParamsPair;
+
 typedef std::tuple<
         ov::Shape,                   // Input 0 Shape
         size_t,                      // Expected num nodes
         size_t,                      // Expected num subgraphs
         std::string                  // Target Device
 > AddConstParams;
 
+typedef std::tuple<
```

Comment:
Seems like useless now... Could you please remove?

Also I see that you add new test `AddSinhPair` which is more comfortable for work with shapes. Maybe we can rewrite previous tests to avoid dupliciation? Maybe not in this PR.. What do you think?

Reply:
1) Removed `AddDynamicParams`
2) Added element type parameter to AddSinhPair to align with other tests
3) Do you mean to rewrite `AddSinh` to `AddSinhPair`? If so, I don't really think that we should do it because `AddSinh` was intentionally written this way to cover all shape combinations to test broadcasting (note that ::testing::Combine covers cartesian product of in_shapes). not sure that it's worth to hard-code 18 test cases in `AddSinhPair` ))